### PR TITLE
Add support for Airport On-Time Performance

### PR DESCRIPTION
**Pull request by tsolakoua**
Solves #69

### CODE REVIEW

File: src/Airport.php [hunk 1]
```diff
@@ -31,12 +32,21 @@ class Airport
      */
     private ?DirectDestinations $directDestinations;
 
+    /**
+     * <p>
+     *   A namespaced client for the
+     *   <code>/v1/airport/predictions</code> endpoints.
+     * </p>
+     */
+    private ?Predictions $predictions;
```
Comment (xianqiliu) [inline src/Airport.php:41]: No need to assign null  ```?``` on must existing endpoint ```$prediction```, see [PR ](https://github.com/amadeus4dev/amadeus-php/pull/73)

File: src/Airport.php [hunk 2]
```diff
@@ -50,4 +60,16 @@ public function getDirectDestinations(): ?DirectDestinations
     {
         return $this->directDestinations;
     }
+
+    /**
+     * <p>
+     *   Get a namespaced client for the
+     *   <code>/v1/airport/predictions</code> endpoints.
+     * </p>
+     * @return Predictions|null
```
Comment (xianqiliu) [inline src/Airport.php:69]: just ```@return Predictions```

File: src/Airport.php [hunk 3]
```diff
@@ -50,4 +60,16 @@ public function getDirectDestinations(): ?DirectDestinations
...
+     */
+    public function getPredictions(): ?Predictions
```
Comment (xianqiliu) [inline src/Airport.php:71]: remove ```?```

File: src/airport/Predictions.php [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+ */
+class Predictions
+{
+    /**
+     * <p>
+     *   A namespaced client for the
+     *   <code>/v1/airport/predictions</code> endpoints.
+     * </p>
+     */
+    private ?OnTime $onTime;
```
Comment (xianqiliu) [inline src/airport/Predictions.php:35]: remove ```?```

File: src/resources/OnTimePrediction.php [hunk 5]
```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Amadeus\Resources;
+
+/**
+ * An OnTimePrediction object as returned by the Airport On-Time Prediction API.
+ * @see OnTimePrediction
```
Comment (xianqiliu) [inline src/resources/OnTimePrediction.php:9]: the leading resource should be redirected to the endpoint which will use this resource, thus it should be ```@see OnTime```

**Comment by xianqiliu** [comment]
@tsolakoua and when we create a new namespace, we should define a mapping from namespaces to directories in ```Composer.json```,  see more details in ```Composer.json```

**Comment by tsolakoua** [comment]
@xianqiliu READY for review!!! 

image

File: src/resources/OnTimePrediction.php [hunk 6]
```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Amadeus\Resources;
+
+/**
+ * An OnTimePrediction object as returned by the Airport On-Time Prediction API.
+ * @see OnTime::getOnTime()
```
Comment (xianqiliu) [inline src/resources/OnTimePrediction.php:9]: ```@see OnTime::get()```

**Comment by xianqiliu** [comment]
@tsolakoua  In order to match the source code, we need to create a folder ```tests/airports/predictions/```, and put the ```OnTimeTest.php``` into ```predictions``` folder

**Review by xianqiliu — APPROVED** [review]

**Comment by xianqiliu** [comment]
@tsolakoua finally! everything looks fine :)

**Pull request merged**